internal/engine/build_and_deployer_live_update_test.go

```diff
@@ -463,3 +468,61 @@ func TestLiveUpdateSomeFilesMatchSyncSomeDontFallsBack(t *testing.T) {
 	}
 	runTestCase(t, f, tCase)
 }
+
+func TestLiveUpdateInFirstImageOfImageDependency(t *testing.T) {
```

Comment:
nice! could you also test that, without an explicit sync step for sancho-base, a changed file in sancho-base prompts a fallback to full build?

Reply:
i like that! done